fbw-a380x/src/systems/instruments/src/MFD/pages/common/DataEntryFormats.tsx

```diff
@@ -3,6 +3,7 @@ import { Subject, Subscribable } from '@microsoft/msfs-sdk';
 import { Mmo, maxCertifiedAlt } from '@shared/PerformanceConstants';
 
 type FieldFormatTuple = [value: string | null, unitLeading: string | null, unitTrailing: string | null];
+type RadioMinimumEntry = 'NONE' | 'NO' | 'NO DH' | 'NODH' | number;
```

Comment:
I'd put this type right above `RadioMinimumFormat`.

Reply:
Moved it 👍